dlc2nwb/utils.py

```diff
@@ -1,16 +1,54 @@
 import cv2
 import datetime
 import os
+from pathlib import Path
 import numpy as np
 import pandas as pd
 import pickle
 import warnings
-from deeplabcut import __version__
-from deeplabcut.utils import auxiliaryfunctions
-from deeplabcut.utils.auxfun_videos import VideoReader
 from hdmf.build.warnings import DtypeConversionWarning
 from pynwb import NWBFile, NWBHDF5IO
 from ndx_pose import PoseEstimationSeries, PoseEstimation
+import yaml
+from ruamel.yaml import YAML
+
+# If available determine version
+try:
+    from deeplabcut import __version__
+    deeplabcut_version = __version__ 
+except:
+    deeplabcut_version = None
+
```

Comment:
I left this here for back compatibility but I think that saving the `__version__` of the installed  deeplabcut version is going to be wrong if the creation is not done immediately after the files are produced. Should't this be on the config file or am I  misunderstanding the purpose of the config file?

Reply:
That's a good point; we wanted to add some more metadata o data files after training/evaluating the network, but haven't done so yet.